comps/llms/text-generation/native/langchain/utils.py

```diff
@@ -42,6 +43,10 @@
 from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
 from transformers.utils import check_min_version
 
+HUGGINGFACEHUB_API_TOKEN = os.getenv("HUGGINGFACEHUB_API_TOKEN", "")
+if HUGGINGFACEHUB_API_TOKEN != "":
+    login(token=HUGGINGFACEHUB_API_TOK EN)
```

Comment:
Looks like an erroneous space?
```suggestion
    login(token=HUGGINGFACEHUB_API_TOKEN)
```